bfvmm/include/vmcs/vmcs_intel_x64.h

```diff
@@ -58,6 +60,9 @@ class vmcs_intel_x64
     ///
     virtual vmcs_error::type launch();
 
+    virtual vmcs_error::type unlaunch();
```

Comment:
I wobbled back and forth on this, I'd be fine with promote (why the doc is missing here)